test/util/malicious/app_test.go

```diff
@@ -55,12 +65,56 @@ func TestOutOfOrderNMT(t *testing.T) {
 	require.NotEqual(t, goodOrderedRoot, root) // quick sanity check to ensure the roots are different
 }
 
-func TestMaliciousNodeTestNode(t *testing.T) {
-	// TODO: flesh out this test further
-	cfg := testnode.DefaultConfig().
-		WithAppCreator(NewAppServer)
+// TestmaliciousTestNode runs a single validator network using the malicious
+// node. This will begin to produce out of order blocks after block height of 5.
+func TestMaliciousTestNode(t *testing.T) {
+	accounts := testfactory.RandomAccountNames(5)
+	cfg := OutOfOrderNamespaceConfig(5).
+		WithAccounts(accounts)
```

Comment:
check out this UX, ehh? We can write our tests like normal, we only have to select a function that pre-configures a testnode.Config for us